### DISCUSSION TITLE
[SVD] Is it possible to perform DDIM inversion on real video using SVD?

### DISCUSSION THREAD

**Discussion by lim142857** [huggingface/diffusers#7187] [Q&A]
1. I tried to replace the SVD's scheduler with DDIM but found that it could not sample normal results. 
2. According to the paper, SVD is trained with continuous noise schedulers, but DDIM is a discrete scheduler.
3. Any suggestions on performing DDIM inversion on real video using SVD? Many video editing methods are based on DDIM inversion.

**Comment by yiyixuxu** [comment]
cc @DN6

**Reply by sayakpaul** [reply]
Cc: @patil-suraj too. 

In order for what you're asking to work I think it might be better to stick to continuous-space noise schedulers and define its own inversion workflow. We recently added the EDM variants here: https://github.com/huggingface/diffusers/pull/7109/. So, you could consider defining its inversion workflow and use it.

**Reply by sayakpaul** [reply]
Cc: @linoytsaban as well in case of any ideas.
